packages/appstore/src/components/modals/real-wallets-upgrade/components/wallet-steps.scss

```diff
@@ -38,6 +38,13 @@
         text-align: center;
         justify-content: center;
 
+        &-points {
+            display: grid;
+            grid-template-columns: 1.6rem 1fr;
+            align-items: center;
+            gap: 0.8rem;
+        }
+
         @include mobile {
```

Comment:
Do we still need this for mobile? If not then just remove it.

Reply:
Yeap we need it cause for mobile its left aligned